doc/xcodebuild.txt

```diff
@@ -0,0 +1,28 @@
+xcodebuild.txt* Plugin to build and test Xcode projects from vim
```

Comment:
I think there's suppose to be a leading `*` here. I feel like I went through this for [investigate](https://github.com/keith/investigate.vim/blob/master/doc/investigate.txt) ages about, but I'm pretty rusty at this point